packages/devp2p/src/rlpx/ecies.ts

```diff
@@ -77,13 +79,7 @@ export class ECIES {
   protected _bodySize: number | null = null
 
   protected _keccakFunction: (msg: Uint8Array) => Uint8Array
-  protected _ecdsaSign: (
-    msg: Uint8Array,
-    pk: Uint8Array,
-  ) => {
-    signature: Uint8Array
-    recid: number
-  }
+  protected _ecdsaSign: Required<CustomCrypto>['ecsign']
```

Comment:
Is this ok from a TypeScript/Node.js compatibility perspective?

(if not AND this was not caught by any unit tests: we should really add this to CI in some form, so that similar code adoptions do not slip through in the future)

Reply:
@gabrocheleau could you take a look here? How would we test this?

Reply:
Same as the other place, there really isn't any issue from a Nodejs compatibility perspective because none of these field declarations are in the compiled code.  Is that what you're asking about?